spec/tictactoe_clojure/computer_player_spec.clj

```diff
@@ -5,31 +5,34 @@
 
 (describe "tictactoe-clojure.computer-player"
   (context "3x3 Board"
-    (with two-move-board ["O" 2 3 "X" 5 6 7 8 9])
-    (with x-1-move-from-win ["O" 2 3 "X" "X" 6 7 8 9])
-    (with o-1-move-from-win ["O" "O" 3 4 5 "X" 7 8 "X"])
-
     (describe "#optimal-move"
-      (it "returns the index of the move with the neutral score if no winning or blocking position available"
-        (should= 5 (optimal-move "O" ["X" 2 3 4 5 6 7 8 9] 1)))
-      (it "returns the index of a move that blocks the opponent from winning"
-        (should= 6 (optimal-move "O" @x-1-move-from-win 1)))
-      (it "returns the index of the move that makes a winning move"
-        (should= 3 (optimal-move "O" @o-1-move-from-win 5)))))
+      (it "makes a winning move for X"
+        (should= 1 (optimal-move "X" [1 "X" "X" "O" "O" 6 7 8 9])))
+      (it "makes a blocking move for X"
+        (should= 1 (optimal-move "X" [1 "O" "O" 4 "X" 6 "X" 8 "X"])))
+      (it "makes a winning move for O"
+        (should= 9 (optimal-move "O" [1 "X" "X" 4 "X" 6 "O" "O" 9])))
+      (it "makes a blocking move for O"
+        (should= 9 (optimal-move "O" ["X" "O" "O" "O" 5 "X" "X" "X" 9])))))
```

Comment:
Could you include the following test case?

```clojure
(it "makes a winning move for X"
        (should= 8 (optimal-move "X" [1 "O" "O" 4 5 "X" "X" 8 "X"])))
```
I'm curious to see if the computer chooses a winning move over a blocking one.